examples/04_glm_first_level/plot_first_level_details.py

```diff
@@ -352,13 +389,16 @@ def plot_contrast(first_level_model):
 )
 first_level_model = first_level_model.fit(fmri_img, events=events)
 z_map = first_level_model.compute_contrast(contrast_val, output_type="z_score")
-plotting.plot_stat_map(
+plot_stat_map(
     z_map,
     display_mode="z",
-    threshold=3.0,
+    threshold=1.96,
```

Comment:
Why do you lower the threshold ? I think it's more reasonable to have something around p<.001 unc.

Reply:
except that doing model selection based on what is significant will most likely lead to overfitting

Reply:
Fair enough, but here we're not doing feature selection. We're simply looking at it.